package.json

```diff
@@ -36,19 +37,22 @@
 	"scripts": {
 		"build": "esbuild src/content-script.ts --bundle --minify --outfile=lib/content-script.js --sourcemap",
 		"compile": "tsc",
+		"firefox:dev": "web-ext run",
+		"firefox:package": "web-ext build",
 		"format": "prettier \"**/*\" --ignore-unknown",
 		"format:write": "yarn format --write",
 		"lint": "npx eslint \"src/*.ts\"  --max-warnings 0 --report-unused-disable-directives",
 		"lint:md": "npx markdownlint \"**/*.md\" \".github/**/*.md\"",
 		"lint:spelling": "cspell \"**\" \".github/**/*\"",
 		"prepare": "husky install",
+		"webext:lint": "web-ext lint",
 		"zip": "node script/zip.js"
 	},
 	"lint-staged": {
 		"*": "prettier --ignore-unknown --write"
 	},
 	"type": "module",
-	"version": "0.2.0",
+	"version": "0.3.0",
 	"engines": {
 		"node": ">=18"
 	}
```

Comment:
[Tooling] https://stackoverflow.com/questions/69526949/yarn-lock-file-resource-source-changed-from-registry-yarnpkg-com-to-registry-npm 🤔 - looks like the `yarn.lock` got changed somehow? Could you try resetting it to what's on `main` and running `yarn` again?

(posting here instead of `yarn.lock` because of github.com PR file viewer performance issues on files with thousands of lines changed)